docs/content/contributing/overview/index.md

```diff
@@ -0,0 +1,43 @@
+---
+type: docs
+title: "Overview: Contributing to Radius"
+linkTitle: "Overview"
+description: "Guides and requirements for contributing to Radius"
+weight: 100
+---
+
+We welcome contributions to Radius! Contributions can come in different ways such as engaging with the community, contributing code, or improving the documentation. This page provides an overview of the different ways you can contribute to Radius.
+
+## Community
+
+Check out the [Radius Community]({{< ref community >}}) page to learn about the different ways you can engage with the Radius community.
+
+## GitHub
+
+If you would like to file Issues, access the source code, or use Codespaces please visit the [Radius GitHub repo](https://github.com/radius-project). 
+
+### good-first-issues
+
+To quickly get started with contributing to code on Radius, here are some identified [good-first-issues](https://aka.ms/radius-first-issues) that you can `/assign` to yourself and start contributing. 
+
+### Contributing to Radius
+
+Check out the following table to learn where and how you can contribute:
+
+| Repository | Description | Contribution guides |
+|------------|-------------|---------------------|
+| **Radius** | Main repository that contains source code for [`rad` CLI]((https://github.com/radius-project/radius/blob/main/docs/contributing/contributing-code/contributing-code-cli/running-rad-cli.md)), [control plane]((https://github.com/radius-project/radius/blob/main/docs/contributing/contributing-code/contributing-code-control-plane/README.md)) and other components of Radius | [radius-project/radius](https://github.com/radius-project/radius/blob/main/CONTRIBUTING.md)|
+| **Docs** | Documentation for Radius | [[Contributing to docs]]({{< ref contributing-docs>}})|
```

Comment:
This has extra brackets: 
<img width="221" alt="image" src="https://github.com/radius-project/docs/assets/54363786/d28a88d1-3888-4076-811c-35cd0ba77cfa">

Reply:
and it doesn't match the pattern of the others

Reply:
thanks for catching..updated the format